packages/graph-explorer/src/connector/openCypher/queries/fetchSchema.ts

```diff
@@ -79,6 +79,7 @@ const fetchVerticesAttributes = async (
       const response = await openCypherFetch<RawVerticesSchemaResponse>(verticesTemplate);
 
       const vertex = response.results[0]?.object as OCVertex;
+      if (!vertex) return;
```

Comment:
nit: avoid adding a return statement, and wrap `vertices.push({` within the condition, to avoid fragmented logic with multiple returns. 
```suggestion
      if (vertex) {
      
      };
```

Reply:
it's return early pattern

Reply:
I agree with @vkagamlyk in this case. Returning early when checking assertions is good practice.

The only nit I would have is to not use inline `if` statements. While it is more code, it is also more readable. It's much easier to find the places in a method that return early if the `return` call is in a predictable place. Having it inline makes the placement depend on the condition expression length.

```ts
if (!vertex) {
  return;
}
```